Composer/packages/client/src/pages/diagnostics/DiagnosticsTable.tsx

```diff
@@ -41,7 +42,8 @@ const itemCount = 10;
 
 export const DiagnosticsTable: React.FC<IDiagnosticListProps> = (props) => {
   const { onItemClick, showType } = props;
-  const diagnostics = useDiagnosticsData();
+  const projectId = useRecoilValue(currentProjectIdState);
```

Comment:
The currentProjectIdState may be mixed the state of design page and diagnostics page 
Can we use the projectId from props directly?
![errors](https://user-images.githubusercontent.com/39758135/109284097-71689080-785a-11eb-9ac1-c7ad2744049e.gif)

Reply:
Fixed. Use props instead of recoilValue `currentProjectId`